.travis.yml

```diff
@@ -38,7 +38,7 @@ install:
 
     # Now do the things we need to do to install it.
     - conda install -c conda-forge --file requirements.txt nose mock python=${PYTHON} ${EXTRA_DEPS} --yes --quiet
-    - if [[ -n ${CONDA_ORIGIN} ]]; then conda install -yq -c ${CONDA_ORIGIN} conda conda-build; fi
+    - if [[ -n ${CONDA_ORIGIN} ]]; then conda install -yq -c ${CONDA_ORIGIN} conda conda-build=2.1; fi
```

Comment:
Hmm...did we loose cb3 compatibility? Admittedly some things in cb3 weren't supported yet ( https://github.com/conda-tools/conda-build-all/issues/97 ), but I thought it did work at least ( https://github.com/conda-tools/conda-build-all/pull/93 ). 😕